pkg/kubectl/cmd/delete.go

```diff
@@ -36,9 +36,22 @@ var (
 	delete_long = templates.LongDesc(`
 		Delete resources by filenames, stdin, resources and names, or by resources and label selector.
 
-		JSON and YAML formats are accepted.
-
-		Only one type of the arguments may be specified: filenames, resources and names, or resources and label selector
+		JSON and YAML formats are accepted. Only one type of the arguments may be specified: filenames,
+		resources and names, or resources and label selector.
+
+		Some resources, such as pods, support graceful deletion. These resources define a default period
+		before they are forcibly terminated (the grace period) but you may override that value with
+		the --grace-period flag, or pass --now to set a grace-period of 1. Because these resources often
+		represent entities in the cluster, deletion may not be acknowledged immediately. If a resource
+		like a pod has not been deleted within the grace period, the node that hosts that pod may be
+		down or unable to communicate to the server. To force delete a resource, you must pass a grace
+		period of 0 and specify the --force flag.
+
+		IMPORTANT: Force deleting pods does not wait for confirmation that the pod's processes have been
+		terminated, which means the pod may still be running on the cluster. This may result in
```

Comment:
I'm just trying not to make this a novel.  Let me see if I can work in a sentence.